runtime/src/accounts_db.rs

```diff
@@ -1307,6 +1313,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
     fn test_accounts_db_serialize() {
```

Comment:
@sambley this is the test that really should work first, then the rest should fall into place, I think